staging/traefik-forward-auth/Chart.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: v1
 appVersion: "latest"
 description: Minimal forward authentication service that provides OIDC based login and authentication for the traefik reverse proxy
 name: traefik-forward-auth
-version: 0.2.14
+version: 0.2.15
```

Comment:
maybe you meant to bump dex chart version instead of tfa? (I don't see any tfa chart changes)

Reply:
nope! thanks.